emsdk/tests/test_emulate.py

```diff
@@ -22,8 +22,7 @@ def test_emulate_function(tmpdir):
         subprocess.run(
             [
                 "emcc",
-                "-s"
-                'BINARYEN_EXTRA_PASSES="--pass-arg=max-func-params@61"',
+                "-s" 'BINARYEN_EXTRA_PASSES="--pass-arg=max-func-params@61"',
```

Comment:
You would want a comma between them; This currently gives string concatenation (but as I mentioned previously I think the entire line can be omitted; otherwise something is pretty wrong with the emsdk patch)